pact-lang-api.js

```diff
@@ -313,23 +313,29 @@ var mkExp = function(pgmName) {
 };
 
 /**
- * Prepare a chainweb-style meta payload.
+ * Prepare a chainweb-style public meta payload.
  * @param sender {string} gas account
  * @param chainId {string} chain identifier
  * @param gasPrice {number} desired gas price
  * @param gasLimit {number} desired gas limit
+ * @param creationTime {number} desired creation time.
```

Comment:
should doco this as "time created (current time) as epoch milliseconds" <-- confirm with @emilypi that this is the intended value

Reply:
Seconds since UNIX epoch.

Reply:
changed to 
```
desired tx's time created (current time) as seconds
```